### ISSUE TITLE
Diagonal (M)ovement doesn't add (D)ragged vehicle penalty

### ISSUE THREAD

**Issue by martosss** [CleverRaven/Cataclysm-DDA#55613]
### Describe the bug

When dragging a cart behind you and moving diagonally, the cart doesn't receive <img src="https://render.githubusercontent.com/render/math?math=(\sqrt{2})"> penalty from diagonal movement.
**Example:**
1) Base move speed - 57(as shown in the top right next to your stats)
2) Movement **non-diagonal with a cart** behind you costs 57 + 70 moves for a total of **127 moves/turn**.
3) Movement **diagonally with a cart is only 150 moves**, which is <img src="https://render.githubusercontent.com/render/math?math=\sqrt{2} * 57  %2B 70"> instead of <img src="https://render.githubusercontent.com/render/math?math=\sqrt{2}*(57 %2B 70)">

### Steps to reproduce

1) spawn a shopping cart with some stuff in it that you can move, but it costs you some extra movement.
2) try to be **fresh**(not weary) and **enable diagonal movement in options**, also don't have any other active effects(although they shouldn't matter).
3) note your walking speed **non-diagonally without** a cart is <img src="https://render.githubusercontent.com/render/math?math=X">
4) note your walking speed **non-diagonally with** a cart is <img src="https://render.githubusercontent.com/render/math?math=X%2BY">
5) note your walking speed **diagonally without** a cart is <img src="https://render.githubusercontent.com/render/math?math=\sqrt{2} * X">
6) note your walking speed **diagonally with** a cart is <img src="https://render.githubusercontent.com/render/math?math=\sqrt{2}X%2BY">

### Expected behavior

in (6)
- your walking speed diagonally with a cart should be <img src="https://render.githubusercontent.com/render/math?math=\sqrt{2}(X%2BY)">, with brackets
- instead of <img src="https://render.githubusercontent.com/render/math?math=\sqrt{2}X%2BY">, without brackets.
 
In other words the cart penalty "<img src="https://render.githubusercontent.com/render/math?math=Y">" should be penalized with a <img src="https://render.githubusercontent.com/render/math?math=(\sqrt{2})"> multiplier

### Versions and configuration

Win 8.1 x64, v0.F-3 Stable, Tiles, English, default mod + soundpack "@"
